functions/src/donation_certificates/createDonationCertificatesFunction.ts

```diff
@@ -4,14 +4,16 @@ import { useFirestore } from '../../../shared/src/firebase/firestoreAdmin';
 import { uploadAndGetDownloadURL } from '../../../shared/src/firebase/storageAdmin';
 import { DonationCertificate, Entity, User } from '../../../shared/src/types';
 import { createDonationCertificateCH } from './pdfGeneration';
+import { sendDonationCertificateMail } from './sendDonationCertificateMail';
 
-export interface CreateDonationCertificatesFunctionProps {
+export interface DonationCertificatesFunctionProps {
 	users: Entity<User>[];
 	year: number;
+	mailFlag: boolean;
 }
 
 export const createDonationCertificatesFunction = functions.https.onCall(
-	async ({ users, year }: CreateDonationCertificatesFunctionProps) => {
+	async ({ users, year, mailFlag }: DonationCertificatesFunctionProps) => {
```

Comment:
Shall we add an auth part here as well (e.g. in a follow-up PR)? Not showing the button in the admin is important for usability, but this function is still publicly callable afaik.

Reply:
Yes, we have to. I have the same issue in my other PR, #232.

Reply:
How should I implement this? Make sure that the email from the token is an email of a global administrator?